gradboost_pv/inference/run.py

```diff
@@ -29,7 +29,9 @@ def process_predictions_to_pandas(predictions: Dict[Hour, Prediction]) -> pd.Dat
                 {
                     "datetime_of_model_inference_utc": pred.datetime_of_model_inference_utc,
                     "datetime_of_target_utc": pred.datetime_of_target_utc,
-                    "forecast_kw": pred.forecast_kw,
+                    "forecast_mw": pred.forecast_mw,
```

Comment:
@peterdudfield I saw that the prod data seems to want the national in MW, so changed that here, just incase it should be different.

Reply:
yea thanks, might have to remove a line from `save.py`

Reply:
I updated that. I didn't see anywhere where the return value is multiplied, so just renamed it.